### DISCUSSION TITLE
Unable to translate file

### DISCUSSION THREAD

**Discussion by zhuoyuejiejie** [SubtitleEdit/subtitleedit#5595] [Q&A]
When I translate the file that appears free API quota exceeded,Is it caused by too many characters in the file?Can you solve this problem?
![1](https://user-images.githubusercontent.com/95971306/145680799-3dc537e5-2f61-475d-bad1-5304a4a767e0.jpg)

**Comment by zebigbos122** [comment]
I think you attempted to translate too many times.
Easily fixed with a VPN and positioning yourself somewhere else.
